api/app/tests/morecast_v2/test_forecasts.py

```diff
@@ -205,37 +223,37 @@ def test_get_forecasts_non_empty(_):
 
 
 def test_construct_wf1_forecast_new():
-    result = construct_wf1_forecast(morecast_record_1,
+    result = construct_wf1_forecast(morecast_input_1,
                                     wfwx_weather_stations,
                                     None,
                                     'test')
-    assert_wf1_forecast(result, morecast_record_1, None, 'test', station_1_url, '1')
+    assert_wf1_forecast(result, morecast_input_1, None, 'test', station_1_url, '1')
 
 
 def test_construct_wf1_forecast_update():
-    result = construct_wf1_forecast(morecast_record_1,
+    result = construct_wf1_forecast(morecast_input_1,
                                     wfwx_weather_stations,
                                     'f1',
                                     'test')
-    assert_wf1_forecast(result, morecast_record_1, 'f1', 'test', station_1_url, '1')
+    assert_wf1_forecast(result, morecast_input_1, 'f1', 'test', station_1_url, '1')
```

Comment:
Perhaps consider updating the `assert_wf1_forecast()` function to test grass curing values